spec/src/test/java/impl/QuestionResourceTest.java

```diff
@@ -34,7 +47,12 @@
 
     @BeforeClass
     public static void before() {
-        testSetup = new TestSetup(postgreSQLContainer);
+        testSetup = new TestSetup(postgreSQLContainer, binder -> binder.bind(SlackResource.class).toInstance(Mockito.mock(SlackResource.class, new org.mockito.stubbing.Answer() {
+            @Override
```

Comment:
maybe a bit repetitive, but i don't know how to solve it

Reply:
Moved it inside the TestSetup